manifests/dconf.pp

```diff
@@ -139,15 +161,25 @@
     profile =>  'gdm',
     path    =>  'org/gnome/login-screen',
     key     =>  'banner-message-text',
-    value   =>  $banner
+    value   =>  $_banner
   }
 
-  # The shutdown button can't be controlled by Dconf.
-  # Use this file type for now to disable it until the policykit module can support the rules.d and javascript methods.
-  file { '/etc/polkit-1/rules.d/10-disable-shutdown-button.rules':
-    owner  => 'root',
-    group  => 'root',
-    mode   => '0644',
-    source => 'puppet:///modules/gnome/10-disable-shutdown-button.rules'
+  if $facts['os']['release']['major'] >= '7' {
+    $_policy_defaults = {
+      ensure   => present,
+      priority => 10,
+      result   => 'yes'
+    }
+
+    polkit::authorization::basic_policy {
+      default:
+        * => $_policy_defaults;
+
+      'Allow anyone to shutdown system':
```

Comment:
@jeannegreulich The underlying `polkit::authorization::rule` defined type makes everything path-safe (hopefully)